WPMedia/fusion-news-theme-blocks
# Themes 1208

## Description

After converting the feature pack to themes V2, some fonts need to have updated to a primary font (Inter). 

- Overline

- Section title links 

## Jira Ticket

- [THEMES-1208](https://arcpublishing.atlassian.net/browse/THEMES-1208)

## Acceptance Criteria

**Actual Behavior**

The overline and the section title links have font primary as (Times) on desktop and mobile
**Expected Behavior**

The overline and the section title links primary font (Inter)

## Test Steps

1. Checkout this branch `git checkout {THEMES-1208}`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/section-title-block`
3. Inspect and check for links and overline font-family

## Dependencies or Side Effects

- Components pr for local development: https://github.com/WPMedia/arc-themes-components/pull/197

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1208]: https://arcpublishing.atlassian.net/browse/THEMES-1208?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
